tests/test_user_controller.py

```
@@ -9,4 +9,5 @@ def test_create_user(test_client: TestClient):
     with test_client as client:
         create_user_dto = UserCreateDTOFactory.build().dict()
         response = client.post("/v1" + USER_CONTROLLER_PATH, json=create_user_dto)
+        print(response.json())
```

print statement

```suggestion
```

---

Thanks for the review - I'm afk right now so will go through everything in more detail later. I just want to jump on this one while you are active. This was inadvertently left in so that I could inspect the failure. How can I get these exceptions to raise rather than return the 500 response?